src/main/java/appeng/integration/IntegrationType.java

```diff
@@ -31,15 +44,37 @@
 
 	MFR( IntegrationSide.BOTH, "Mine Factory Reloaded", "MineFactoryReloaded" ),
 
-	Waila( IntegrationSide.BOTH, "Waila", "Waila" ),
+	Waila( IntegrationSide.BOTH, "Waila", "Waila" )
+	{
+		@Override
+		public IIntegrationModule createInstance()
+		{
+			return new WailaModule();
+		}
+	},
 
 	InvTweaks( IntegrationSide.CLIENT, "Inventory Tweaks", "inventorytweaks" ),
 
-	JEI( IntegrationSide.CLIENT, "Just Enough Items", "JEI" ),
+	JEI( IntegrationSide.CLIENT, "Just Enough Items", "JEI" )
+	{
+		@Override
+		public IIntegrationModule createInstance()
+		{
+			return Integrations.setJei( new JEIModule() );
+		}
+	},
 
 	Mekanism( IntegrationSide.BOTH, "Mekanism", "Mekanism" ),
 
-	OpenComputers( IntegrationSide.BOTH, "OpenComputers", "OpenComputers" );
+	OpenComputers( IntegrationSide.BOTH, "OpenComputers", "OpenComputers" ),
+
+	THE_ONE_PROBE( IntegrationSide.BOTH, "TheOneProbe", "theoneprobe" ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)